.konflux/must-gather/must-gather.konflux.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather-rhel9:v4.19 as mgbuilder
+FROM brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather-rhel9:v4.20@sha256:dec17bbfe2171444e97b45a13e30eae306adc734ca1648817098937ff686191e as mgbuilder
```

Comment:
Is this image allowed in the current conforma policy?

Reply:
passing the `confroma on pr ` isn't enough here ?

Reply:
@rbaturov  when we onboard must gather in a different PR lets check the image again.  for now this is prep work

Reply:
interetsing indeed, I don't see it is allowed in the relaxed policy, ant not in standard policies either https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy/registry-telco-5g-integrationtest.yaml?ref_type=heads how come the conforma passed?

Reply:
there is no conforma currently set for must-gather dockerfile.  as i mentioned - this is prep work